test/plugins/features/movies/controller.test.js

```diff
@@ -12,16 +12,145 @@ describe('movie controller', () => {
 
       return Controller.create(payload)
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(payload.title);
 
         return new Movie({ id: movie.id}).fetch();
 
       })
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(payload.title);
       });
     });
 
   });
-  
+
+  describe('retrieve a list of movies', () => {
+
+    it('returns all movies', () => {
+      let count;
+      return new Movie().fetchAll()
+      .then((movies) => {
+        return movies.length;
```

Comment:
Nit: You could set `count` here and return `Controller.findAll` in the same promise, removing the need for an additional `.then`. This goes for the other tests here.